### ISSUE TITLE
logical replication between 2 data centers

### ISSUE THREAD

**Issue by Eric-zch** [CrunchyData/postgres-operator#4056]
Hi There,
Assume I have a Crunchy postgrescluster in Kubernetes cluster 1 and another postgrescluster in Kubernetes cluster 2. Can I set up logical replication from Data Center 1 to Data Center 2?

I read this [link](https://access.crunchydata.com/documentation/postgres-operator/latest/guides/logical-replication) Crunchy Data Documentation which seems to be about logical replication within the same Kubernetes cluster.

**Comment by andrewlecuyer** [comment]
@Eric-zch, yes that should be possible.  The same concepts in the documentation you referenced can be applied to PostgresCluster's in two different Kubernetes clusters.  You will simply need to ensure your database is accessible across Kubernetes cluster boundaries.
